### DISCUSSION TITLE
🚀 Feature: KolAvatar with dynamic background colors

### DISCUSSION THREAD

**Discussion by Chrisdo82** [public-ui/kolibri#6776] [Ideas]
### Goals

1. to be able to change the background color dynamically

### Non-Goals

1.
2.
3.

### Background

current alternative is to create it by ourself

### Proposal

it should be implemented as in the badge

**Comment by deleonio** [comment]
Hi @Chrisdo82, we will discuss this feature in our team and give your feedback.

**Comment by deleonio** [comment]
Behandeln wir als ticket weiter: https://github.com/public-ui/kolibri/issues/8612
